test-resources-localstack/test-resources-localstack-s3/src/main/java/io/micronaut/testresources/aws/localstack/s3/LocalStackS3Service.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2017-2021 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.testresources.aws.localstack.s3;
+
+import io.micronaut.testresources.aws.localstack.LocalStackService;
+import org.testcontainers.containers.localstack.LocalStackContainer;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Adds support for Localstack S3.
+ */
+public class LocalStackS3Service implements LocalStackService {
+
+    private static final String AWS_S3_ENDPOINT_OVERRIDE = "aws.s3.endpoint-override";
```

Comment:
This path will be deprecated
```suggestion
    private static final String AWS_S3_ENDPOINT_OVERRIDE = "aws.services.s3.endpoint-override";
```